test/fixtures/orderbook.js

```diff
@@ -372,6 +372,35 @@ module.exports.QUALITY_OFFERS = [
   }
 ];
 
+module.exports.INCORRECT_TAKER_PAYS_FUNDED_OFFERS = [
```

Comment:
Why is this called `INCORRECT_TAKER_PAYS_FUNDED_OFFERS`? What is incorrect about it?